### DISCUSSION TITLE
Save TextField in the Database NOT IdField from Dropdown

### DISCUSSION THREAD

**Discussion by juliermefelix** [serenity-is/Serenity#6394] [Help] [answered]
I have a table called RedeSocial (id, networkname) using LooKupEditor I am feeding a field (String) in another table, in this field I want the text to be recorded and not the ID.
How do I write text field and not ID field?

**Comment by wezmag** [comment, marked as answer]
You can create a RedeSocialLookup extends RowLookupScript.
In the constructor, assign networkname to both IdField and TextField.

for example
``` csharp
public class RedeSocialLookup : RowLookupScript<RedeSocialRow>
{
   public RedeSocialLookup()
   {
      IdField = TextField = RedeSocialRow.Fields.NetworkName.PropertyName;
   }
}
```
After create this lookup, use it on the field of the "another table" form.
``` csharp
[LookupEditor(typeof(RedeSocialLookup))]
public string NameOfTheField { get; set; }
```
